common/tests/test_simple_page.py

```diff
@@ -38,6 +39,7 @@ def setUpTestData(cls):
 
     def setUp(self):
         super(SimplePageStatisticsTagsTestCase, self).setUp()
+        self.home_page = HomePage.objects.first()
```

Comment:
I've noticed on our re-design branch these tests are skipped bbc2c72#diff-e1ece5f3301b936378213bf662c40a7b048dcfdd8bebb8501fa857a18e4d6cb3R17

I'm a bit puzzled at the moment how this test was passing when self.home_page wasn't defined confused

Reply:
I mentioned this in the meeting, but for posterity, we're marked a bunch of tests as skip on `redesign` because we removed all templates, which broke a bunch of tests.